### ISSUE TITLE
Need to log a message if List has more than 1 element

### ISSUE THREAD

**Issue by harshadloya** [bazaarvoice/jolt#920]
I am using java springboot application. In that, jolt spec is applied on the message to transform it.
While looping over an array, how can I log a message if I have more than 1 element found?

For Example: 
Input:
{
  "employees": [
    {
      "fname": "ABC",
      "lname": "XYZ"
    },
    {
      "fname": "ABC",
      "lname": "JKL"
    },
    {
      "fname": "AAA",
      "lname": "MMM"
    }
  ]
}

Spec:
[
  {
    "operation": "shift",
    "spec": {
      "employees": {
        "*": {
          "fname": {
            "ABC": {
              "@2": {
                "fname": "Names.FirstName",
                "lname": "Names.LastName"
              }
            }
          }
        }
      }
    }
  }
]

Output:
{
  "Names" : {
    "FirstName" : [ "ABC", "ABC" ],
    "LastName" : [ "XYZ", "JKL" ]
  }
}

**Comment by zeshuai007** [comment]
@harshadloya Hi. I don't understand what you expect.
Observing your specification, you have converted the array into a message.
Would you like to convert FirstName: ["ABC", "ABC"] to FirstName: ["ABC"], remove duplicates like this?

**Comment by harshadloya** [comment]
No, I would like to achieve something like below java code snippet output

Snippet:
```
for(int i=0; i<array.length;i++) {
     if(i>0){
          log.info("More than 1 element found");
     }
}
```

Output:
If array.length = 5
"More than 1 element found"  will be printed to the console 4 times.

**Comment by zeshuai007** [comment]
@harshadloya The output you expect will be helpful if it can be expressed in json format
